packages/cli/src/commands/env/pull.ts

```diff
@@ -48,7 +48,8 @@ export default async function pull(
   environment: ProjectEnvTarget,
   opts: Partial<Options>,
   args: string[],
-  output: Output
+  output: Output,
+  cwd: string = process.cwd()
```

Comment:
```suggestion
  cwd: string,
```
This shouldn't be optional. We should provide the `--cwd` global flag here, right?

Reply:
The global `--cwd` flag sets the process's current directory before the command functions are invoked. https://github.com/vercel/vercel/blob/e40eecafc97dbe120f4c95f30b7b9aaf621e4b81/packages/cli/src/index.ts#L139-L142

I needed an override here for tests only.